feconf.py

```diff
@@ -252,6 +252,11 @@ def get_empty_ratings():
 # merely avoids checking for whether the promo bar is enabled for every Oppia
 # page visited.
 ENABLE_PROMO_BAR = True
+# Whether to enable maintenance mode on the site. For non-admins, this redirects
+# all HTTP requests to the maintenance page. This is the only check which
+# determines whether the site is in maintenance mode to avoid queries to the
+# database by non-admins.
+ENABLE_MAINTENANCE_MODE = False
```

Comment:
Do we want this switch to be in the web admin interface so that we don't have to re-deploy to change this? (I'm only asking this since I'm unfamiliar with code mods in GAE production)